content/4_Gold/Unordered_Map.mdx

```diff
@@ -146,6 +146,53 @@ However, this hash function is quite bad; if we insert
 $(0,0), (1,1), (2,2) \ldots$ then they will all be mapped to the same bucket (so
 it would easily be **hacked**).
 
+<LanguageSection>
+
+<CPPSection />
+
+<JavaSection>
+
+A better (and easier) way to hash custom objects is to use Java's built in 
+`Objects.hash()` method. This method takes in multiple objects and uses them
+to create a hash code.
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class HashTest {
+	public static void main(String[] args) {
+		// uses custom hash function in class Pair
+		Set<Pair> set = new HashSet<>();
+	}
+
+	static class Pair {
+		public int f, s;
+
+		public Pair(int f, int s) {
+			this.f = f;
+			this.s = s;
+		}
+
+		@Override
+		public int hashCode() {
+			return Objects.hash(f, s);
+		}
+
+		@Override
+		public boolean equals(Object o) {
+			if (!(o instanceof Pair)) return false;
```

Comment:
braces

Reply:
i feel like doing it on one line is more concise, esp given that this line isnt the main focus of the code

Reply:
actually
maybe wrap this in a codesnip if that's the problem

Reply:
well the method is still important because i talk about how java needs the equals method to work

Reply:
if it's important then what's the problem with expanding it a bit?

Reply:
is there a need to expand it? what's the problem with being a bit more concise, esp when the stuff in the if block is so short

Reply:
i mean it makes it easier for others to "play" with the code